tests/test_templates.py

```diff
@@ -106,6 +106,13 @@ def test_rule_bindings_collection_team_3(self):
             Path("tests/resources/configs/rule_bindings/team-3-rule-bindings.yml")
         )
 
+    @pytest.fixture
+    def test_rule_bindings_collection_team_4(self):
+        """ """
+        return lib.load_rule_bindings_config(
+            Path("tests/resources/configs/rule_bindings/team-4-rule-bindings.yml")
+        )
+
```

Comment:
is this fixture being used anywhere?

Reply:
I would also suggest adding the not blank rule to an existing test fixture instead of a new one, e.g.

`
rule_bindings:
  T2_DQ_1_EMAIL:
    entity_id: TEST_TABLE
    column_id: VALUE
    row_filter_id: DATA_TYPE_EMAIL
    rule_ids:
      - NOT_NULL_SIMPLE
      - REGEX_VALID_EMAIL
      - CUSTOM_SQL_LENGTH_LE_30
      - NOT_BLANK
    metadata:
      brand: one
`

You will most likely need to edit other test resources e.g. the expected json and generated sql.

Reply:
Unit Tested it separately as a single data quality rule.